site/.eleventy.js

```diff
@@ -1,17 +1,18 @@
-const fs = require('fs');
-const color = require('kleur');
+import fs from 'fs';
+import kleur from 'kleur';
```

Comment:
Cosmetic: it was more readable and useful (in case of migration) to use the alias `color` because it is exactly why we use this dependency.